### PR TITLE
SAK-48663 css: add underline back to links in some tools

### PR DESCRIPTION
**Pull request by adrianfish**
https://sakaiproject.atlassian.net/browse/SAK-48663

Fixes announcements and rwiki. Also, fixes a hover issue with the site links in the tools navigation on the left.

### CODE REVIEW

File: library/src/skins/default/src/sass/modules/tool/portal/_portal.scss [hunk 1]
```diff
@@ -288,6 +291,13 @@ a[href]:not(.btn):hover {
 
   .site-link-block:hover {
     background-color: var(--site-nav-btn-hover-bg);
+    color: var(--site-nav-btn-current-hover-color);
+    a {
+      color: inherit;
+    }
+    i {
+      color: inherit;
+    }
```
Comment (kunaljaykam) [inline library/src/skins/default/src/sass/modules/tool/portal/_portal.scss:300]: We can make it more concise? 
```suggestion

  a, i {
    color: inherit;
  }
```

File: rwiki/rwiki-tool/tool/src/webapp/WEB-INF/vm/info.vm [hunk 2]
```diff
@@ -340,7 +340,7 @@
 				       <input type="submit" name="updatePermissions" value="${rlb.jsp_button_save}" />
 				       <input type="hidden" name="realm" value="${currentRWikiObject.realm}" />
 				       #if (${realmBean.siteUpdateAllowed})
-				         <span>${rlb.jsp_edit_page_permissions_msg1} <a href="${util.escapeHtml(${realmBean.editRealmUrl})}" >${rlb.jsp_edit_site_permissions2}</a></span>
+				         <span>${rlb.jsp_edit_page_permissions_msg1} <a href="${util.escapeHtml(${realmBean.editRealmUrl})}">EGGs ${rlb.jsp_edit_site_permissions2}</a></span>
```
Comment (kunaljaykam) [inline rwiki/rwiki-tool/tool/src/webapp/WEB-INF/vm/info.vm:343]: EGGs?

Comment (adrianfish) [inline rwiki/rwiki-tool/tool/src/webapp/WEB-INF/vm/info.vm:343]:
```suggestion
				         <span>${rlb.jsp_edit_page_permissions_msg1} <a href="${util.escapeHtml(${realmBean.editRealmUrl})}">${rlb.jsp_edit_site_permissions2}</a></span>
```

**Review by kunaljaykam — APPROVED** [review]
GTG!, Thanks @adrianfish

**Pull request merged**